rule_packages/c/IO1.json

```diff
@@ -88,7 +88,7 @@
       },
       "queries": [
         {
-          "description": "",
+          "description": "Close files when they are no longer needed",
```

Comment:
being picky but some of these are missing the recommended [full stop](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description) - "."